cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -167,7 +168,9 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    time.sleep(35)
```

Comment:
After validation, we have to wait for approx.30-35 sec.  for credentials to be validated from docker.io. Tried all possible conditions which can be use in `wait_for` but not showing required output hence `time.sleep` is added.

Reply:
@sbulage please add bz, or explanation that we discussed

Reply:
@rlbabyuk yeah sure, I will add it after discussion with devel.